cypress/integration/validation/profile.js

```diff
@@ -2,6 +2,11 @@ const { debug } = require("console")
 
 describe('Checking Profile', () => {
     beforeEach(() =>{
+        cy.task("queryDb",
+        `UPDATE toolbox.users
+        SET username = "test_user", email = "max@test.test", password = "$2y$10$UJinKiSroTM2MsFmLBu2puwOJynkzmTj9ZgLnsOHfa3HIK43/QKxW"
+        WHERE id = 1`)
+
         cy.loginViaApi("max@test.test", "password")
```

Comment:
shouldn't this login data retrieved by env variables?  (loginViaApi and in update statement